careers/blocks/header/header.js

```diff
@@ -377,13 +329,15 @@ export default async function decorate(block) {
     nav.setAttribute('aria-expanded', 'false');
 
     // prevent mobile nav behavior on window resize
-    toggleMenu(nav, navSections, screenConfig.smallDesktop.media.matches);
+    toggleMenu(nav, navSections, screenConfig.smallDesktop.media.matches
+      || screenConfig.largeDesktop.media.matches);
 
     // add event listeners when window width changes
     screenConfig.smallDesktop.media.addEventListener('change', () => {
       const localNavTitle = block.querySelector('.local-nav-title');
       const globalNavTitle = block.querySelector('.global-nav-title');
-      toggleMenu(nav, navSections, screenConfig.smallDesktop.media.matches);
+      toggleMenu(nav, navSections, screenConfig.smallDesktop.media.matches
```

Comment:
Why do we need this? The CSS should just handle hiding / showing the mobile nav vs regular nav based on media queries.